README.md

```diff
@@ -28,21 +34,21 @@ npm run storybook
 npm run build
 ```
 
-### Publishing Packages
+## Publishing Packages
 
-#### Setting up your npm account
+### Setting up your npm account
 
 - Create an npm account with your bbc email address. https://www.npmjs.com
 - In your npm profile settings, set up Two Factor Authentication. Enable it for authorization and publishing
 - Back up your recovery codes for your account
 - Join the BBC npm org by following the process here: https://github.com/bbc/npm When you're added to the org and signed in, you should be able to see all public and private `@bbc` packages here: https://www.npmjs.com/settings/bbc/packages
 
-#### Publishing a package
+### Publishing a package
 
 - In your terminal, run `npm login` and follow the steps for authentication
 - `npm publish packages/<PATH_TO_PACKAGE> --otp=<YOUR_2FA_CODE>`
   e.g. `npm publish packages/components/psammead-paragraph --otp=<YOUR_2FA_CODE>`
 
-#### Making a package public
+### Making a package public
```

Comment:
This can be done in a single command with `--access public`, might be simpler

Reply:
Hmm agreed this command is simpler, but then we'd have to add a line about `cd`ing into the package directory, whereas the current command works from the root level (as does the one above that). So for consistency I'd suggest we stick with what we have.

Reply:
Oh no soz I meant doing 
```
npm publish packages/components/psammead-paragraph --access public --otp=<YOUR_2FA_CODE>
```
Not overly bothered though